web/react/pages/channel.jsx

```diff
@@ -70,8 +69,8 @@ function setupChannelPage(props) {
     // Modals
     //
     ReactDOM.render(
-        <GetLinkModal />,
-        document.getElementById('get_link_modal')
+        <GetTeamInviteLinkModal />,
```

Comment:
We did this in our apps using [React-Portal](https://github.com/tajo/react-portal), so you can place the Modal inside the context that triggered it rather than having to put it all the way up the tree to the root just so it renders properly.